packages/ndla-ui/src/Article/ArticleSideBar.tsx

```diff
@@ -44,15 +50,34 @@ type Props = {
   onLinkToResourcesClick?: (e: React.MouseEvent<HTMLElement>) => void;
   copyPageUrlLink?: string;
   licenseBox?: React.ReactNode;
+  resourcesRef?: any;
```

Comment:
Vi prøver å bruke `any` typen så lite som mulig. 
Tror den kan erstattes med `RefObject<HTMLElement>` i dette tilfellet:
```suggestion
  resourcesRef?: RefObject<HTMLElement>;
```